CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED]
 
+### Changed
+
+- Removed `dask_executor_tests.py` to the dask plugin repo
+
```

Comment:
Should use the `Tests` tag instead of changed and `dask_executor_tests.py` has been moved to the dask plugin repo, not removed.